python/pyspark/ml/tests.py

```diff
@@ -371,6 +378,126 @@ def test_fit_maximize_metric(self):
         self.assertEqual(1.0, bestModelMetric, "Best model has R-squared of 1")
 
 
+class RegressorTest(PySparkTestCase):
```

Comment:
Each trainer output corresponding models. So we cannot share training code but creating DataFrames can be separated each `RegressorTest` and `ClassificationTest`. I'll update so.